test/helpers/dao.js

```diff
@@ -0,0 +1,62 @@
+const { hash } = require('eth-ens-namehash')
+const { getEventArgument } = require('@aragon/contract-test-helpers/events')
+
+const Kernel = artifacts.require('@aragon/core/contracts/kernel/Kernel')
+const ACL = artifacts.require('@aragon/core/contracts/acl/ACL')
+const EVMScriptRegistryFactory = artifacts.require(
+  '@aragon/core/contracts/factory/EVMScriptRegistryFactory'
+)
+const DAOFactory = artifacts.require(
+  '@aragon/core/contracts/factory/DAOFactory'
+)
+
+const newDao = async rootAccount => {
+  // Deploy a DAOFactory.
```

Comment:
We should definitely move this to a `@aragon/contract-helpers-test-dao` package or similar.